.lando.yml

```
@@ -7,6 +7,22 @@ config:
   database: mariadb:10.4
   xdebug: false
 services:
+  # Replaces default lando MariaDB image with arm64 (Apple M1) compatible one.
+  # TODO: remove database service overrides once lando is updated.
+  database:
+    type: compose
+    services:
+      image: mariadb:latest
+      command: docker-entrypoint.sh mariadbd
+      restart: always
+      ports:
+        - '3306'
+      environment:
+        MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 'true'
+        MARIADB_DATABASE: drupal9
+        MYSQL_DATABASE: drupal9
+        MARIADB_USER: drupal9
+        MARIADB_PASSWORD: drupal9
```

I think we could get rid of the `db-export` and `db-import` warnings by adding these.
```suggestion
        MARIADB_PASSWORD: drupal9
        LANDO_EXTRA_DB_EXPORT_ARGS: '--protocol=TCP'
        LANDO_EXTRA_DB_IMPORT_ARGS: '--protocol=TCP'
```

---

Where are you finding these answers?

---

https://github.com/lando/cli/blob/main/plugins/lando-services/scripts/sql-export.sh
https://github.com/lando/cli/blob/main/plugins/lando-services/scripts/sql-import.sh

---

> I think we could get rid of the `db-export` and `db-import` warnings by adding these.

Confirmed that these lines remove the warnings.

---

Actually, I found these weren't needed when using `mariadb:10.4` on line 15.